packages/shared/charts/src/rollup/index.js

```diff
@@ -0,0 +1,150 @@
+const { join } = require("path");
```

Comment:
Most of this file is copied over from the existing versions, except that I had to change imports to require to avoid rollup issues